### PR TITLE
feat: use Link component rather than anchor tag

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
## Related Issue

Closes: #511 

### Describe the changes you've made

- Use `<Link>` by replacing `<a>` in entire codebase to improve performance

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: backup src/components/Home/Projects.jsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { Link } from "lucide-react";
```
Comment (mkubdev) [inline backup src/components/Home/Projects.jsx:1]:
```suggestion
import Link from "next/link";
```

ig it's the correct import

**Review by mkubdev — CHANGES_REQUESTED** [review]
Thanks a lot to have took the time to update all missing <Link> !
There is a wrong import in `Project` component, but LGTM!

**Comment by mkubdev** [comment]
> I've noticed that some changes have been made to the backup source, which are unnecessary. This backup contains the previous version of the website and doesn't require these modifications.

Yeah and I'm thinking of removing it. Maybe keeping in on a branch, but we shouldn't save the backup on main, wdyt ?

**Review by mkubdev — APPROVED** [review]

**Comment by vinzvinci** [comment]
> Yeah and I'm thinking of removing it. Maybe keeping in on a branch, but we shouldn't save the backup on main, wdyt ? I'm switching to approve change.

Yes, good thing to separate on another branch.

File: backup src/components/Home/Projects.jsx (hunk 1 above)
Comment (Anmol-Baranwal) [reply backup src/components/Home/Projects.jsx]: Done.

**Review by vinzvinci — APPROVED** [review]

**Review by s2sharpit — APPROVED** [review]
LGTM!

**Pull request merged**